CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 2.2.4 - 2022-05-25
+- Change most output types to non-optional. Classified as a bug fix. Filtered outputs are still optional!
```

Comment:
if varscan is meant to be semantically versioned, isn't this a backwards-compatibility-breaking change?

Reply:
I doubt it. Though right now having all optional outputs prevents varscan from running at all.